test/e2e/s3/cross-account.sh

```diff
@@ -48,7 +48,6 @@ cat <<EOF | kubectl apply -f -
 apiVersion: s3.services.k8s.aws/v1alpha1
 kind: Bucket
 metadata:
-  namespace: testing
```

Comment:
Hmm, this seems unrelated. Can you comment why this change was necessary?

Reply:
When we run the test for s3 service, at that time this resource look for testing namespace which is not present in out testing or we are not creating any such namespace in our testing script anywhere due to which test case for s3  was falling. SO fix this is I have removed this line so that Bucket resource can be created in default namespace or select name.

Reply:
Ah, OK.

@A-Hilaly, unless I'm mistaken, the cross-account resource management e2e test *requires* a Kubernetes `Namespace` to be created (and subsequently annotated), right? Was this e2e test assuming someone had already created a 'testing' `Namespace`? If so, we need to modify the test script to create the 'testing' `Namespace` and ensure it is annotated appropriately.

Reply:
Exactly, I already had a 'testing' `Namespace` on my cluster and i missed script the steps to create it..

> If so, we need to modify the test script to create the 'testing' Namespace and ensure it is annotated appropriately.

Yes, we will ideally need an annotated `namespace` to test the override/default account/region features